docs/user-guide/api.md

```diff
@@ -1,17 +1,36 @@
-# API Design
+# API
 
-Our API was designed with three principles in mind:
+Screwdriver APIs and the data models around them are documented via [Swagger]. This prevents out-of-date documentation, enables clients to be auto-generated, and most importantly exposes a human-readable interface.
 
-1. All interactions of user's data should be done through the API, so that
-there is a consistent interface across the tools (CLI, Web UI, etc.).
-1. Resources should be ReST-ful and operations should be atomic so that intent
-is clear and human readable.
-1. API should be versioned and self-documented, so that client code generation
-is possible.
+> **Version 4** is the current API, all links should be prefixed with `/v4`.
+
+See our [API documentation].
+
+## Using the API
+### With Swagger
+Swagger documentation includes examples and editable parameters to play around with. Visit the `/documentation` page and use the interactive `Try it out!` buttons to make calls to our API.
```

Comment:
We call out that the API is at `v4` at the beginning of the page, but I can add it to all the routes as well
And as for exposing our hosted API endpoint, I think we want users to look at their own hosted API for reference, since it'll change depending on what docker image their API is using. I can say something like "here's our API for reference" and add the link to `api.screwdriver.cd/v4/documentation`.